src/test/java/com/sproutsocial/nsq/PublisherWithFailoverDockerTestIT.java

```diff
@@ -59,6 +61,22 @@ public void withBackup_failoverAndFailBackProactivly() {
         sendAndVerifyMessagesFromPrimary(publisher, handler);
     }
 
+    @Test
+    public void failedBulkPublishesPublishToFailoverSequentially() {
+        final int maxBodySize = 5242880; // This is the default nsqd max body size for any nsqd TCP message
+        final int count = 30;
+        final int bytesPerMessage = (maxBodySize + 10) / count;
```

Comment:
Nit: Truncation could cause an issue here(even though its probably not).  Can we do this instead to make 100% sure we are over size

```suggestion
        final int bytesPerMessage = (maxBodySize + 10) / (count-1);
```